tests/services/orders/validation_service/workflows/balsamic/test_rules.py

```diff
@@ -103,3 +104,42 @@ def test_double_normal_samples_in_case(
     # THEN the error should concern the double tumours in the case
     assert isinstance(errors[0], DoubleNormalError)
     assert errors[0].case_index == 0
+
+
+def test_normal_only_wgs_in_case_with_new_sample(valid_order: BalsamicOrder, base_store: Store):
+
+    # GIVEN that the sample in the order is WGS and normal
+    valid_order.cases[0].samples[0].application = "WGSPCFC030"
+    valid_order.cases[0].samples[0].tumour = False
+
+    # WHEN validating that the order contains only one sample that is normal and WGS
+    errors: list[NumberOfNormalSamplesError] = validate_number_of_normal_samples(
+        order=valid_order, store=base_store
+    )
+
+    # THEN an error should be returned
+    assert errors
+
+    # THEN the error should concern that case having only one sample that is normal and WGS
+    assert isinstance(errors[0], NormalOnlyWGS)
+    assert errors[0].case_index == 0
+
+
+def test_normal_only_wgs_in_case_with_existing_sample(
+    valid_order_with_existing_sample: BalsamicOrder, store_with_existing_sample: Store
+):
+
+    # GIVEN an order with an existing sample and a store with the corresponding sample
+    # GIVEN that the sample in normal and WGS
+
+    # WHEN validating that the order contains only one sample that is normal and WGS
```

Comment:
```suggestion
    # WHEN validating if the order contains only one sample that is normal and WGS
```
Very minor but it sounds as if you want it to contain a normal WGS sample.

Reply:
Yes!